cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/messages/StreamEntity.java

```diff
@@ -40,8 +40,13 @@
 	public StreamEntity()
 	{
 	}
-	
-	
+
+	public StreamEntity(UUID instanceId, String streamName)
+	{
+		this.instanceId = instanceId;
+		this.streamName = streamName;
```

Comment:
streamName is declared above as @PartitionKey while CassandraTablesCreator defines it "withClusteringColumn",